disruption_generator/experiments/example.yaml

```diff
@@ -2,13 +2,13 @@
 - disrupt_action:
     - name: Kill services on VM start
```

Comment:
I see it for testing - change name for "Kill .." to testing ping to file or something like that.